bedrock-entry.sh

```diff
@@ -94,6 +94,14 @@ if [ ! -f "bedrock_server-${VERSION}" ]; then
   mv bedrock_server bedrock_server-${VERSION}
 fi
 
+if [ -n "$WHITE_LIST" ]; then
+  echo "Setting whitelist"
+  rm -rf whitelist.json
+  echo $WHITE_LIST | awk -v RS=, 'BEGIN{print "["}; {print "{ \"name\": \"" $1 "\" },"}; END{print "]"}' > whitelist.json
```

Comment:
Per my comment on the helm chart PR https://github.com/itzg/minecraft-server-charts/pull/21/files#r512119253 it does look like these are actually XUIDs and usernames:

https://minecraft.gamepedia.com/Whitelist.json

Reply:
Yes, I ran this on a bedrock server and tested. Whitelist allows name or xuid, but permissions are xuid-only.

Reply:
Lookup would be nice, but requires an api account on microsoft servers. You can't hit an open endpoint and look things up. I agree that it would be nice to do this for people, but I'm not sure that it can be done that easily.

Reply:
Yep, that's as far as I got :)